helm/values.yaml

```diff
@@ -12,7 +12,9 @@ image:
 imagePullSecrets: {}
 
 containerPort: 9012
-containerHealthProbePort: 9013
+containerPortName: grpc-port
+containerAdminPort: 9013
+containerAdminPortName: http-admin
```

Comment:
Context? I don't have any particular attachment to the names, but why do we need any port names here when we're binding by port numbers anyway?

Reply:
Trying to have consistent port-names so that it'll be easier to parse from the yaml in the [debugging toolkit](https://github.com/Traceableai/debugging-toolkit) we are building.